ServiceDesk.BLL/Services/TicketService.cs

```
@@ -24,16 +28,35 @@ public async Task<Status> GetTicketStatus(Guid id, CancellationToken cancellatio
         return ticket?.Status ?? throw new NullReferenceException("Ticket is null");
     }
 
-    public async Task SetTicketStatus(Guid id, Status newStatus, CancellationToken cancellationToken)
+    public async Task<TicketModel?> SetTicketStatus(Guid id, Status newStatus, CancellationToken cancellationToken)
     {
         var ticket = await _ticketRepository.GetEntityByIdAsync(id, cancellationToken);
 
         if (ticket == null)
         {
-            return;
+            return null;
```

It will be better to throw an exception here